scipy/stats/_stats_py.py

```diff
@@ -7214,15 +7226,11 @@ def chisquare(f_obs, f_exp=None, ddof=0, axis=0):
     -----
     This test is invalid when the observed or expected frequencies in each
     category are too small.  A typical rule is that all of the observed
```

Comment:
```suggestion
    This test is not well-calibrated when the observed or expected frequencies in each
    category are too small.  A typical rule is that all of the observed
```
Perhaps more precise here? "Invalid" doesn't have a statistical meaning.

Reply:
I see "invalid" quite often in the statistics literature.

The main violation is that the size (rejection frequency under the null) is not maintained, if assumption do not hold.

It's less precise than other terms, but we don't need to specify what exactly will go wrong under different violations of assumptions.

Reply:
You're right, "invalid" is used; apologies - I meant that it doesn't have a precise meaning.  And, it implies that tests are either "valid" or "not valid"; when in practice most tests (including this one) are approximate - it's a case of "all models are false but some are useful". How about 
```
This test is not accurate ...
```
... ?

One reason I'm suggesting this is because it's very common for people to ask things like "is it allowed to do this test"; as if that's a yes-no question, rather than asking "how useful will this be".

Reply:
I like "not well-calibrated" although that implies that the reader knows what "calibration" means in this context, so I am also fine with the more lax term "invalid" or "not accurate".